drivers/ps2/Kconfig.npcx

```diff
@@ -0,0 +1,24 @@
+# NPCX PS2 configuration options
+
+# Copyright (c) 2021 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig PS2_NPCX
+	bool "Nuvoton NPCX embedded controller (EC) PS2 driver"
+	depends on SOC_FAMILY_NPCX && ESPI_PERIPHERAL_8042_KBC
+	help
+	  Enable the NPCX family PS2 driver. The driver also
```

Comment:
PS/2

Should describe what it supports (keyboard?) and what it does. Needs more detail

Reply:
Done.